tethysext/atcore/templates/atcore/resource_workflows/components/spatial_dataset_form.html

```diff
@@ -41,6 +44,7 @@ <h6 class="title">{{ dataset_title }}</h6>
                         <td>
                             {% with is_numeric=column_is_numeric|return_item:column %}
                             <input {% if is_numeric %}type="number"{% else %}type="text"{% endif %}
+                                {% if is_numeric %}step="0.001"{% endif %}
```

Comment:
I know we agreed to make this hard-coded, but let's add this as another option on the step so the user can override the default value of 0.001.